pkg/cmd/server/api/validation/validation.go

```diff
@@ -82,6 +84,19 @@ func ValidateHTTPServingInfo(info api.HTTPServingInfo) fielderrors.ValidationErr
 	return allErrs
 }
 
+func ValidateDisabledFeatures(disabledFeatures []string, field string) fielderrors.ValidationErrorList {
+	allErrs := fielderrors.ValidationErrorList{}
+
+	fl := api.FeatureList{}
+	for i, feature := range disabledFeatures {
+		if err := fl.Add(feature); err != nil {
```

Comment:
This works, but it's hard to read without knowing the rest of the code.  Compare the `feature` against the `knownFeatures` directly to make it obvious what's happening via inspection.